sigar_interface.go

```diff
@@ -301,6 +301,19 @@ type NetRawV6ConnList struct {
 	List []NetConn
 }
 
+type ProcessList struct {
+	List []Process
+}
+
+type Process struct {
+	ProcState
+	ProcIo
+	ProcMem
+	ProcTime
+	ProcArgs
+	ProcExe
+}
```

Comment:
That's correct - this technique is how Go does <a href="https://dave.cheney.net/2015/05/22/struct-composition-with-go">struct composition</a>.